### ISSUE TITLE
Bed Aura

### ISSUE THREAD

**Issue by Thunder33345** [ImpactDevelopment/ImpactClient#216]
Adding a separate module bedaura 

Bedaura will be active in end/nether(unless configured to use in overworld for whatever reasons(server plugins etc)), which it will use the bed on behalf of client thus causing the explosion of the bed(also delay timer config for less sensitive servers)

Should also add a bed replacer, which moves bed from inventory into your hand if you placed a bed using said slot, since beds are not stackable which may runs out quickly and having to stop to(which someone might want to use even if bed aura is off)

**Comment by LeafHacker** [comment]
This should probably also blow up end crystals and maybe tnt, with all
targets being toggle-able.

**Comment by ZeroMemes** [comment]
Duplicate of #214

**Comment by LeafHacker** [comment]
@ZeroMemes this isn't a duplicate of that issue. That issue is about breaking beds (e.g  defensively) while this one is about using them (and potentially about triggering other explosives too)

**Comment by wholesomeduck** [comment]
So basically breaker, but specifically for beds? Because breaker is already a module.

**Comment by LeafHacker** [comment]
@wholesomeduck no, that is #214 which was already added in 4.2. This issue is about automatically _using_ any bed in range. Potentially after calculating the damage it would inflict on the player and any nearby entities.

**Comment by leijurv** [comment]
Could be adapted from #872

**Comment by dominikaaaa** [comment]
Changed milestone to 4.9 as per #1751
